conf/services/check-transit-guarantee-balance.yml

```diff
@@ -0,0 +1,24 @@
+serviceName: Check your transit guarantee balance
+serviceHeaderName: Check your transit guarantee balance
+serviceDescription:
+  The Common Transit Convention (CTC) is a treaty between EU member states and other countries for the transit of goods.
+  The goods can pass through different customs regimes without them having to be declared at every border.
+  This service allows users to check the balance of their guarantee so that they know how much of their balance is available to use.
+serviceDomain: www.tax.service.gov.uk
+serviceUrl: /check-guarantee-balance
+contactFrontendServiceId: http://tax.service.gov.uk/check-guarantee-balance/eori-number
+complianceStatus: partial
+accessibilityProblems:
+  - Switching to the Welsh language version does not translate the messaging. It displays unreadable text.
+  - The continue waiting button does not allow users to wait but takes users to the next page in their journey.
+milestones:
+  - description:
+      Missing Welsh translations of the content causes rendering eligible copy when user switches to Welsh language version. 3.1.1 Language of page.
```

Comment:
Again, this may be able to come out as the fix has been implemented but a re-test is awaiting. Need to speak to Yetty about this.